docs/admin/extensible-admission-controllers.md

```diff
@@ -195,7 +193,7 @@ To turn it on, you need to
 
 ### Write a webhook admission controller
 
-See [here](https://github.com/caesarxuchao/example-webhook-admission-controller)
+See [caesarxuchao/example-webhook-admission-controller](https://github.com/caesarxuchao/example-webhook-admission-controller)
```

Comment:
@lavalamp you might want to take a look at my example. I'll consolidate it a little bit more tonight.